torch/_inductor/graph.py

```diff
@@ -148,6 +149,61 @@ def getattr_recursive(obj, target):
     return attr_itr
 
 
+def mark_nodes_dislike_padding(g):
+    """
+    Nodes like convolution/convolution_backward want its input to be contiguous.
```

Comment:
nit: s/contiguous/dense

Reply:
Hmm, what would be the difference between 'dense' and 'contiguous'? I see convolution/convolution_backward has calls like `tensor.contiguous(memory_format=the_format)`

Reply:
Sometimes contiguous is taken to assume `torch.contiguous(memory_format=torch.contiguous_format)`. Whereas dense is more clear it might be channels last or contiguous. It doesn't really matter, either is fine.

Reply:
Got it. Make sense to me!